src/components/ChatLog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const entries = props.entries;
+
+  const entryComponents = entries.map((entry, i) => {
+    return (
+      <div className="chat-log" key={entry.id}>
+        <ChatEntry
+          id={entry.id}
+          sender={entry.sender}
+          body={entry.body}
+          timeStamp={entry.timeStamp}
+          liked={entry.liked}
+          updateHeartColorCallback={props.updateHeartColorCallback}
+          odd={i % 2}
```

Comment:
This works, but it depends on the messages being in a specific order.  If we were pulling this information from a database, there might be more complexity, such as a user sending two messages before the other person responded.  I recommend using something like a test on the sender here to determine how to display the message.